src/navigators/createMaterialTopTabNavigator.js

```diff
@@ -17,16 +18,35 @@ type Props = InjectedProps & {
   tabBarPosition?: 'top' | 'bottom',
   tabBarComponent?: React.ComponentType<*>,
   tabBarOptions?: TabBarOptions,
+  lazyOnSwipe: boolean,
+  sceneAlwaysVisible: boolean,
```

Comment:
Don't think this should be exposed either. Loading tabs in between could block JS thread and result in poor UX. And as far as I have seen, no one really wants this behavior.